.github/workflows/test_linux.yaml

```diff
@@ -0,0 +1,40 @@
+name: test_ubuntu
+
+# ensure only one job runs for this repo and OS
+concurrency:
+  group: ${{ github.ref }}-ubuntu
+  cancel-in-progress: true
+
+on:
+  push:
+  pull_request:
```

Comment:
Are we sure we want to run this unrestricted? Specially in this repo there's a lot of stuff that's not raku-related...